ISISNeutronMuon/MDANSE
# Split plotting gui

**Description of work**
A prototype of the new plotting interface has been added as two separate tabs.

**Fixes**
1. It is possible to overplot results originating from different files.
2. Physical unit switching is possible using combo boxes in the plot tab.
3. Single, Grid and Heatmap plotting modes are available at the moment.
4. Sliders can be used to modify the plot contents. Their function depends on the plotting mode.

**To test**
Load several files containing data analysis results.
Try plotting the results in a single plot.
Change the plotting mode and the physical units to check if the changes are applied correctly.
